generated/nidaqmx/_library_interpreter.py

```diff
@@ -1758,12 +1754,14 @@ def create_table_scale(
                         ctypes_byte_str]
 
         error_code = cfunc(
-            name, prescaled_vals, len(prescaled_vals), num_prescaled_vals_in,
-            scaled_vals, len(scaled_vals), num_scaled_vals_in,
-            pre_scaled_units, scaled_units)
+            name, prescaled_vals, len(prescaled_vals), scaled_vals,
+            len(scaled_vals), pre_scaled_units, scaled_units)
         check_for_error(error_code)
 
     def create_task(self, session_name):
+        task = lib_importer.task_handle(0)
+        new_session_initialized = True
```

Comment:
the `new_session_initialized` parameter has been handled separately in the library interpreter generation and this is always set to `True`